x-pack/solutions/observability/plugins/infra/public/alerting/inventory/components/metric.tsx

```diff
@@ -99,20 +107,12 @@ export const MetricExpression = ({
   popupPosition,
   nodeType,
 }: Props) => {
-  const [popoverOpen, setPopoverOpen] = useState(false);
+  const [popoverOpen, { toggle: togglePopover }] = useBoolean(false);
```

Comment:
This solves the following problem:

![popover_problem](https://github.com/user-attachments/assets/d46de324-e8ca-4baf-8f6f-ae285f137d69)